sdk/identity/azure-identity/samples/azure-aad-auth-with-redis-py.md

```diff
@@ -0,0 +1,141 @@
+## Azure Cache for Redis: Azure AD with redis client library
+
+### Table of contents
+
+- [Dependency requirements](#dependency-requirements)
```

Comment:
Replace this with 
`- [Prerequisites](#prerequisites)`

Add the section as follows on top of document:

#### Prerequisites
-  Configuration of Role and Role Assignments is required before using the sample code in this document.
 - **Dependency Requirements:**
    - [redis](https://pypi.org/project/redis/)
    - [azure-identity for Python](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/identity/azure-identity)